pepper-apis/src/main/java/org/broadinstitute/ddp/export/json/structured/DsmComputedRecord.java

```diff
@@ -10,6 +10,8 @@
 
     @SerializedName("dateOfBirth")
     private String dateOfBirth;
+    @SerializedName("dateOfMajority")
+    private String dateOfMajority;
```

Comment:
I like that we're putting DoM into the computed record! I think that's a good place to put it.